public/docs/_examples/toh-5/dart/lib/heroes_component.dart

```diff
@@ -26,7 +26,11 @@ class HeroesComponent implements OnInit {
   Hero selectedHero;
 
   // #docregion renaming
-  HeroesComponent(this._heroService, this._router);
+  HeroesComponent(this._heroService,
```

Comment:
It's technically OK in TS although misleading and especially w/o the trailing `)`.  Not loving it. There should be another way to convey the message.